packages/camera/camera/example/android/app/build.gradle

```diff
@@ -21,18 +22,15 @@ if (flutterVersionName == null) {
     flutterVersionName = '1.0'
 }
 
-apply plugin: 'com.android.application'
-apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
-
 android {
     namespace 'io.flutter.plugins.cameraexample'
     compileSdk flutter.compileSdkVersion
 
 
     defaultConfig {
         applicationId "io.flutter.plugins.cameraexample"
-        minSdkVersion flutter.minSdkVersion
-        targetSdkVersion 28
+        minSdkVersion 21
```

Comment:
Isn't this undoing what @stuartmorgan just did in https://github.com/flutter/packages/pull/8035?

Reply:
yup nice catch. done.